frontend/hub/namespaces/HubNamespaceForm.tsx

```diff
@@ -142,6 +144,9 @@ function HubNamespaceInputs(props: { isDisabled?: boolean; isRequired?: boolean
         label={t('Logo URL')}
         placeholder={t('Enter logo URL')}
       />
+      <PageFormSection singleColumn>
+        <PageFormMarkdown<HubNamespace> label={t('Markdown')} name="resources" />
```

Comment:
Markdow is very generic name...at the end we are adding resources.  Perhaps, we should call `Resources`?

Also, from the old UI we had a footnote...


![image](https://github.com/ansible/ansible-ui/assets/9053044/1a032f5c-68fe-4e5b-bb89-4879ba214cc4)